commands/mta_command_test.go

```diff
@@ -186,39 +194,85 @@ var _ = Describe("MtaCommand", func() {
 	})
 })
 
-func getGetAppsModel(name, state string, runningInstances, totalInstances int,
-	memory, diskQuota int64, host, domain string) plugin_models.GetAppsModel {
-	return plugin_models.GetAppsModel{
-		Name:             name,
-		State:            state,
-		RunningInstances: runningInstances,
-		TotalInstances:   totalInstances,
-		Memory:           memory,
-		DiskQuota:        diskQuota,
-		Routes: []plugin_models.GetAppsRouteSummary{
-			plugin_models.GetAppsRouteSummary{
-				Host: host,
-				Domain: plugin_models.GetAppsDomainFields{
-					Name: domain,
-				},
-			},
+func getApps(name, state string) []models.CloudFoundryApplication {
+	return []models.CloudFoundryApplication{
+		{
+			Name:  name,
+			Guid:  "app-guid",
+			State: state,
 		},
 	}
 }
 
-func getGetServicesModel(name, offering, plan, opType, opState string, boundApplications []string) plugin_models.GetServices_Model {
-	return plugin_models.GetServices_Model{
-		Name: name,
-		Service: plugin_models.GetServices_ServiceFields{
-			Name: offering,
-		},
-		ServicePlan: plugin_models.GetServices_ServicePlan{
-			Name: plan,
+func getProcessStats(runningInstances, totalInstances int64, memory, diskQuota int64) []models.ApplicationProcessStatistics {
+	var processes []models.ApplicationProcessStatistics
+	for i := int64(0); i < runningInstances; i++ {
```

Comment:
int64(0) isn't that an overkill :D?

Reply:
Compilation fails when this is absent because it can't compare an untyped integer (`i`) and an int64 (`runningInstances`)